packages/ariakit/src/form/__examples__/form/test.tsx

```diff
@@ -55,7 +56,7 @@ test("show error on submit", async () => {
 
 test("focus on input with error on submit", async () => {
   render(<Example />);
-  await click(getByRole("button"));
+  await click(getAllByText("Submit")[0]!);
```

Comment:
```suggestion
  await click(getByRole("button", { name: "Submit" }));
```

Reply:
Cool, wasn't sure how to handle this. LGTM